Source/Core/Common/StringUtil.cpp

```diff
@@ -77,13 +77,10 @@ bool AsciiToHex(const std::string& _szValue, u32& result)
   // Set errno to a good state.
   errno = 0;
 
-  char* endptr = nullptr;
-  const u32 value = strtoul(_szValue.c_str(), &endptr, 16);
+  const u32 value = strtoul(_szValue.c_str(), nullptr, 16);
 
-  if (!endptr || *endptr)
```

Comment:
I assume the function is supposed to fail if the value is partially converted. So these checks shouldn't be removed.